store/transfer_processor.go

```diff
@@ -169,17 +169,29 @@ func (s *transferProcessor) process(ctx context.Context, events []*model.Transfe
 			to := gethCommon.BytesToAddress(e.To)
 			contractsAddrs[contractAddr][to] = struct{}{}
 		}
-	}
 
-	// Make sure coinbases are also included in balance queries to geth
-	for _, addr := range coinbases {
-		if allSubs[addr] == nil {
-			continue
-		}
-		if contractsAddrs[model.ETHAddress] == nil {
-			contractsAddrs[model.ETHAddress] = make(map[gethCommon.Address]struct{})
+		if e.IsMinerRewardEvent() {
+			to := gethCommon.BytesToAddress(e.To)
+			reward, _ := new(big.Int).SetString(e.Value, 10)
+
+			if len(minerRewardDiff) > 1 {
+				s.logger.Error("Detect two miner reward events on block", "block", e.BlockNumber, "coinbase", common.BytesTo0xHex(e.To), "balance", e.Value)
```

Comment:
maybe find a way to log the old one too?

Reply:
add new struct function

```
func (s *transferProcessor) printUnexpectedRewardEvent(e *model.Transfer, rewardDiff map[gethCommon.Address]*big.Int) {
	for addr, value := range rewardDiff {
		s.logger.Error("Previous event diff", "block number", e.BlockNumber, "from", string(e.From), "to", addr.Hex(), "balance", value)
	}

	s.logger.Error("Current event diff", "block number", e.BlockNumber, "from", string(e.From), "to", string(e.To), "balance", e.Value)
}

```